packages/expo/e2e/__tests__/logout-test.ts

```diff
@@ -0,0 +1,55 @@
+/* eslint-env jest */
+import fs from 'fs/promises';
+
+import { execute, getLoadedModulesAsync, projectRoot } from './utils';
+
+const originalForceColor = process.env.FORCE_COLOR;
+
+beforeAll(async () => {
+  await fs.mkdir(projectRoot, { recursive: true });
+  process.env.FORCE_COLOR = '1';
+});
+afterAll(() => {
+  process.env.FORCE_COLOR = originalForceColor;
+});
+
+it('loads expected modules by default', async () => {
+  const modules = await getLoadedModulesAsync(`require('../../build-cli/cli/logout');`);
+  expect(modules).toStrictEqual([
+    'node_modules/ansi-styles/index.js',
+    'node_modules/arg/index.js',
+    'node_modules/chalk/source/index.js',
+    'node_modules/chalk/source/util.js',
+    'node_modules/has-flag/index.js',
+    'node_modules/supports-color/index.js',
+    'packages/expo/build-cli/cli/log.js',
+    'packages/expo/build-cli/cli/logout/index.js',
+    'packages/expo/build-cli/cli/utils/args.js',
+    'packages/expo/build-cli/cli/utils/errors.js',
+  ]);
+});
+
+it('runs `npx expo logout --help`', async () => {
+  const results = await execute('logout', '--help');
+  expect(results.stdout).toMatchInlineSnapshot(`
```

Comment:
I remember some kind of clean up function from another PR to remove the special characters. removeAnsiSomething...? Use here for consistency? (And in other similar test above and below)

Reply:
https://github.com/expo/expo/blob/main/packages/expo/cli/utils/__tests__/utils.ts#L1